paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/Expression.java

```diff
@@ -239,6 +251,36 @@ public String eval(String input) {
         }
     }
 
+    /** date format from a time input. */
+    final class DateFormat implements Expression {
+
+        private static final long serialVersionUID = 1L;
+
+        private final String fieldReference;
+        private final DateTimeFormatter formatter;
```

Comment:
Is this Serializable? Maybe you just need a lazy object here.

Reply:
Thank you for your suggestion, i resubmitted the code .